docs/quickstart/learning-more-lakefs.md

```diff
@@ -82,21 +73,26 @@ This command will mount your present working directory (PWD) within the containe
 
 To use lakeFS with MinIO (or other S3-compatible object storage), use the following example:
 
-Note: Make sure the Quickstart Docker Compose from the previous steps isn't also running as you'll get a port conflict.
-{: .note .note-info }
-
    ```bash
 docker run --pull always -p 8000:8000 \
-   -e LAKEFS_BLOCKSTORE_TYPE='s3' \
-   -e LAKEFS_BLOCKSTORE_S3_FORCE_PATH_STYLE='true' \
-   -e LAKEFS_BLOCKSTORE_S3_ENDPOINT='http://<minio_endpoint>' \
-   -e LAKEFS_BLOCKSTORE_S3_DISCOVER_BUCKET_REGION='false' \
-   -e LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID='<minio_access_key>' \
-   -e LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY='<minio_secret_key>' \
+   -e LAKEFS_BLOCKSTORE_TYPE=s3 \
+   -e LAKEFS_BLOCKSTORE_S3_FORCE_PATH_STYLE=true \
+   -e LAKEFS_BLOCKSTORE_S3_ENDPOINT=http://<minio_endpoint> \
+   -e LAKEFS_BLOCKSTORE_S3_DISCOVER_BUCKET_REGION=false \
+   -e LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID=<minio_access_key> \
+   -e LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY=<minio_secret_key> \
    treeverse/lakefs run --local-settings
    ```
 
   </div>
+  <div class="ui-tabs-panel">
+
+>Notes:
+1. Multi line commands are in unix syntax. For windows - please edit commands before execution
```

Comment:
This is inconsistent with what we've done above. Either we support Windows throughout the docs, or we don't. 

```suggestion
1. Multi line commands are in Unix syntax. For Windows - please edit commands before execution
```

> please edit commands

If we're saying this we need to tell the user what to edit them to, otherwise we put on them the assumption that they know what to change.